automatic/curl/README.md

```diff
@@ -0,0 +1,7 @@
+# <img src=""></img> [curl](https://chocolatey.com/packages/curl)
```

Comment:
yup, svg, png, jpg and ico (The last two was added since a package or two used them).